### PR TITLE
Fix/449 tier1 identity stop gradient bias add

### PR DESCRIPTION
**Pull request by khatchad**
Found some missing work.

### CODE REVIEW

File: com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflow2Model.java [hunk 1]
```diff
@@ -6433,6 +6433,74 @@ public void testConvertToTensor()
...
+  @Test
+  public void testStopGradientKwarg()
+      throws ClassHierarchyException, IllegalArgumentException, CancelException, IOException {
+    test("tf2_test_stop_gradient_kwarg.py", "f", 1, 1, Map.of(2, Set.of(TENSOR_3_FLOAT32)));
+  }
+
+  /**
+   * Tier-1 generator (wala/ML#449): {@code tf.nn.bias_add(value, bias)} returns a fresh tensor with
+   * the same shape and dtype as {@code value} (bias is broadcast-added but doesn't change the
+   * receiver's shape).
```
[the remark this answers is not in the document]
Comment (khatchad) [reply com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflow2Model.java:8340]: Valid observation, but this Javadoc is pre-existing on `master` from [#202](https://github.com/ponder-lab/ML/pull/202) — the merge in this PR brings it back into the diff view but doesn't introduce it. You're right that "Tier-1 generator" overstates the implementation: `bias_add` is currently modeled as `<new>+<return>` in `tensorflow.xml` (line 2161-2168), routing through `convert_to_tensor` rather than a dedicated `BiasAdd.java` generator. Tightening the Javadoc (or adding the dedicated generator) is out of scope for this small kwarg-tests salvage; worth a follow-up PR.

File: com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/Identity.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  }
+
+  /**
+   * Constructs an {@code Identity} anchored to a manual node. Used by {@link
+   * TensorGenerator#createManualGenerator}.
+   *
+   * @param node The {@link CGNode} for the {@code identity.do()} synthetic method.
+   */
+  public Identity(CGNode node) {
+    super(node);
```
[the remark this answers is not in the document]
Comment (khatchad) [reply com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/Identity.java:64]: Pre-existing on `master` from [#202](https://github.com/ponder-lab/ML/pull/202); the merge here brings it into the diff but it's not new in this PR. The critique is correct: `Identity` is dispatched in `TensorGeneratorFactory.getGenerator(source, builder)` (line 1294) but not in `TensorGenerator.createManualGenerator(node, builder)`, so the `CGNode`-taking constructor is currently unreachable. Worth a follow-up to either wire it into the manual dispatch or delete the unreached constructor + its Javadoc reference.

File: com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/Identity.java [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    /** The input tensor. */
+    INPUT;
+
+    public String getName() {
+      return name().toLowerCase();
+    }
+
+    public int getIndex() {
+      return ordinal();
+    }
```
[the remark this answers is not in the document]
Comment (khatchad) [reply com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/Identity.java:45]: This is already addressed — `Parameters.getName()` on line 40 uses `name().toLowerCase(Locale.ROOT)`, and `java.util.Locale` is imported on line 12. The fix landed in [`c498cddd`](https://github.com/ponder-lab/ML/commit/c498cddd) ("Address Copilot review: Javadoc accuracy on the three flagged nits") on the original PR #202 branch and was squash-merged. Looks like the reviewer is seeing the pre-fix version of the file.

File: com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/StopGradient.java [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+   * Constructs a {@code StopGradient} anchored to a manual node. Used by {@link
+   * TensorGenerator#createManualGenerator}.
+   *
+   * @param node The {@link CGNode} for the {@code stop_gradient.do()} synthetic method. (Pre-fix
+   *     this could also be the {@code read_data()} synthetic, but the new XML model uses {@code
+   *     <new>+<return>} in {@code do()} directly — no {@code read_data} entry point.)
+   */
+  public StopGradient(CGNode node) {
+    super(node);
+  }
```
[the remark this answers is not in the document]
Comment (khatchad) [reply com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/StopGradient.java:70]: Same as the [Identity.java thread](https://github.com/ponder-lab/ML/pull/324#discussion_r3290692484) — pre-existing on `master` from [#202](https://github.com/ponder-lab/ML/pull/202), not introduced by this PR. `StopGradient` is dispatched in `TensorGeneratorFactory` (line 1295-1296) but not in `createManualGenerator`; the `CGNode` constructor is unreachable. Worth a follow-up.

File: com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/StopGradient.java [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    /** The input tensor. */
+    INPUT;
+
+    public String getName() {
+      return name().toLowerCase();
+    }
+
+    public int getIndex() {
+      return ordinal();
+    }
```
[the remark this answers is not in the document]
Comment (khatchad) [reply com.ibm.wala.cast.python.ml/source/com/ibm/wala/cast/python/ml/client/StopGradient.java:48]: Same as the [Identity.java locale thread](https://github.com/ponder-lab/ML/pull/324#discussion_r3290692500) — already uses `Locale.ROOT` on line 43 (`java.util.Locale` imported on line 12), as of [`c498cddd`](https://github.com/ponder-lab/ML/commit/c498cddd).
